File: namsor/ruby/src/ApiKeyInfoExample.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["X-API-KEY"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::AdminApi.new.api_key_info

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling AdminApi#api_key_info: #{e}"
end

File: namsor/typescript-node/src/ApiKeyInfoExample.ts
import * as fs from 'fs';
import api from "namsor_client"
import models from "namsor_client"

const apiCaller = new api.AdminApi();
apiCaller.setApiKey(api.AdminApiApiKeys.api_key, "YOUR_API_KEY");

apiCaller.apiKeyInfo().then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling AdminApi#apiKeyInfo:");
  console.log(error.body);
});
